projects/demo/src/modules/components/combo-box/examples/6/index.ts

```diff
@@ -25,7 +25,7 @@ const STRINGIFY_EMPLOYEE: TuiStringHandler<Employee> = (item: Employee) =>
     providers: [tuiItemsHandlersProvider({stringify: STRINGIFY_EMPLOYEE})],
 })
 export class TuiComboBoxExample6 {
-    readonly testValue = new UntypedFormControl(null);
+    readonly testValue = new FormControl(null);
```

Comment:
```suggestion
    readonly testValue = new FormControl<Employee | null>(null);
```

(it not only about TypeScript; it can be cool to show (for documentation's reader) that control can get this value)